wiki/Gameplay/Fingerlock/de.md

```diff
@@ -0,0 +1,29 @@
+---
+stub: true
+tags:
+  - wrist lock
+  - wristlock
+  - wrist-lock
+  - finger-lock
+  - finger lock
+---
+
+# Fingerlock
+
+*Nicht zu verwechseln mit [Notelock](/wiki/Gameplay/Notelock) oder [Mind block](/wiki/Gameplay/Mind_block).*
+
+**Fingerlock** (auch `Finger-Lock` oder `Finger Lock` genannt) ist ein Begriff, der das Gefühl beschreibt, die Kontrolle über die Finger zu verlieren, während man osu! spielt. Obwohl der Begriff nicht spezifisch für osu! ist, wird er von Spielern häufig verwendet, um ihre Frustration über Situationen auszudrücken, in denen ein Spieler weiß, wann und wo er ein [Hit-Objekt](/wiki/Hit_object) treffen muss, aber unerklärlicherweise die Taste einfach nicht drückt, was zu einem Gefühl führt, als sei der Finger "gesperrt" (engl. "locked"). Daher auch der Name "Fingerlock".
```

Comment:
What a massive sentence there. I can't find any alternative myself lol

Reply:
yeah, its just as massive in the english one, so i guess its fine?